src/framework/preferences/js/Enactors.js

```diff
@@ -310,6 +315,13 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                 "model.value": "value"
             }
         },
+        styles: {
+            enabled: "fl-textSize-enabled"
+        },
+        cssCustomProp: {
+            factor: "--fl-textSize-factor",
```

Comment:
Similar to --fl-lineSpace-factor I don't see that this is used in a CSS definition